src/content/guides/deploying-istio/index.md

```diff
@@ -0,0 +1,283 @@
+
+- Ingress / Egress gateways 
+- Sidecar injector webhook
+- Galley
+- Mixer
+- Pilot
+- Grafana
+- Prometheus
+- Tracing (Jaeger) 
+- Kiali
+- Mutual TLS
+- Certmanager
+
+All of those are deployed but mutual TLS and certmanager. You can configure the components tweaking the values passed to helm. Run `helm inspect install/kubernetes/helm/istio` to display the different options available.
+
+Although Istio can run in almost every Kubernetes cluster no matter the provider, there are some quirks to consider depending on where the service mesh is deployed.
+
+### AWS and Azure
+
+```
+$ helm install install/kubernetes/helm/istio --name istio --namespace istio-system
+```
+
+### Giantnetes
+
+```
+$ helm install install/kubernetes/helm/istio --name istio --namespace istio-system \
+--set gateways.istio-ingressgateway.type=NodePort --set gateways.istio-egressgateway.type=NodePort 
+```
+
+Ensure all components are up and running listing the pods in the istio namespace.
+```
+$ kubectl get pod -n istio-system -w
+```
+
+## Deploy an application
+
+Let's verify Istio is deployed and configure correctly. We can deploy a simple http server that returns `200` code for the `healthz` path of the service.
+
+### Automated sidecar injection
+
+First of all we need to label the namespace to make easier the sidecar injection.
+```
+kubectl label namespace default istio-injection=enabled
+```
+And now we can install the chart containing the example app
+```
+$ helm registry install quay.io/giantswarm/liveness-chart
+```
+
+### Manual sidecar injection
+
+In case you want the manual injection, pull the chart and use istio command line to parse the deployment and insert the proxy.
+```
+# Pulling the chart to your local
+$ helm registry pull quay.io/giantswarm/liveness-chart --dest /tmp/
+
+# (CLOUD) Parse the chart template to Kubernetes resources
+$ helm template /tmp/liveness/giantswarm_liveness-chart_XXX/liveness-chart /tmp/liveness.yaml
+
+# (GIANTNETES) Parse the chart template to Kubernetes resources (use the ingress port given by support)
+$ helm template /tmp/liveness/giantswarm_liveness-chart_XXX/liveness-chart /tmp/liveness.yaml --set ingress.port=30010
+
+# Inject the sidecar in the deployment and submit the resources to Kubernetes API
+$ istioctl kube-inject -f /tmp/liveness.yaml | k apply -f -
+```
+
+It will launch a deployment and service listening in the port 80. At the same time, it will create a virtual service redirecting the traffic to the liveness service. Also, there is a gateway to wire the virtual service up with the ingress gateway.
+
+### AWS verification
+
+Get service hostname given by the cloud provider. Run
+```
+$ kubectl get svc istio-ingressgateway -n istio-system -o jsonpath="{.status.loadBalancer.ingress[0].hostname}"
```

Comment:
`kubectl get svc` -> `kubectl get service` (use explicit standard naming in favour of abbreviation)